src/main/java/com/apkspectrum/plugin/PackageSearcherLinker.java

```diff
@@ -37,6 +39,10 @@ public void launch() {
 			name = ApkInfoHelper.getResourceValue(labels,
 					getPreferLangForAppName());
 			break;
+        	case SEARCHER_TYPE_PACKAGE_HASH :
```

Comment:
Please set the indentation to tab.
I plan to change the indentation method to spaces in the future, but
Please set it as a tab until it changes.

Reply:
Oops, I didn't check the indentation. Thanks for your correction.